mmv1/templates/terraform/examples/workbench_instance_full.tf.erb

```diff
@@ -72,7 +72,7 @@ resource "google_workbench_instance" "<%= ctx[:primary_resource_id] %>" {
 
   disable_proxy_access = "true"
 
-  instance_owners  = [ "<%= ctx[:test_env_vars]["service_account"] %>"]
+  instance_owners  = "example@example.com"
```

Comment:
I think the VCR error goes here: "Inappropriate value for attribute "instance_owners": list of string required."

Reply:
Fixed it to use a list.